### DISCUSSION TITLE
Waybar wallbash problem

### DISCUSSION THREAD

**Discussion by mahaveergurjar** [prasanthrangan/hyprdots#1169] [Q&A]
Waybar wallbash problem after upgrade to latest Hyprdots the waybar can't fetch the wallbash color
![240324_16h54m05s_screenshot](https://github.com/prasanthrangan/hyprdots/assets/57872034/f427d33f-321f-47de-9892-a99a6161d459)
![240324_16h59m09s_screenshot](https://github.com/prasanthrangan/hyprdots/assets/57872034/37b47a96-bfd0-4721-94ac-a95a9039b923)

**Comment by kRHYME7** [comment]
``` Hyprdots cache rebuild ``` ? 

from titttu

> ... Waybar won't load if the .dcol cache file is not upto date

**Comment by mahaveergurjar** [comment]
cache rebuild but waybar not work also reload cache  but not working wallbash
yes after update the wallbash by @prasanthrangan

**Reply by prasanthrangan** [reply]
Hi, could you check if `swwwallbash.sh` is upto date?
also check if the rgba variables are available in `~/.config/swww/wall.dcol` -

```
❯ head ~/.config/swww/wall.dcol
dcol_pry1="312F50"
dcol_pry1_rgba="rgba(49,47,80,\1)"
dcol_txt1="FFFFFF"
dcol_txt1_rgba="rgba(255,255,255,\1)"
dcol_1xa1="2B2952"
dcol_1xa1_rgba="rgba(43,41,82,\1)"
dcol_1xa2="3D3A6B"
dcol_1xa2_rgba="rgba(61,58,107,\1)"
dcol_1xa3="4E4B7D"
dcol_1xa3_rgba="rgba(78,75,125,\1)"
```

**Comment by mahaveergurjar** [comment]
ok
